source/change-streams/change-streams.rst

```diff
@@ -575,6 +548,21 @@ A possible interface for this callback MAY look like:
 
 This MUST NOT be implemented in synchronous drivers. This MAY be implemented in asynchronous drivers.
 
+Updating the Cached Resume Token
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This method returns the cached ``resumeToken`` following these rules:
```

Comment:
Reword this section. "This method returns" doesn't apply anymore. This describes how we update the cached `resumeToken`.